sql/mozfun/norm/get_windows_info/README.md

```diff
@@ -1,19 +1,19 @@
-# Windows Names, Versions, and Builds
+### Windows Names, Versions, and Builds
```

Comment:
Why not H1 here? This seems to be the top level header in this specific file?

Reply:
As documented in the commit message:

> mkdocs' ToC generator will stop when the header level goes up again.
> Because the UDF name itself is generated as a first level heading, any
> UDF with a first-level header documentation will thus stop rendering any
> subsequent headers.
> Most notably on /mozfun/hist where only the very first UDF got a ToC
> entry.